src/pudl/metadata/fields.py

```diff
@@ -4299,6 +4384,10 @@
         # TODO: Disambiguate as this means different things in different tables.
         "description": "Physical street address.",
     },
+    "street_address_2": {
+        "type": "string",
+        "description": "Secondary street address.",
+    },
```

Comment:
We already have an `address_2` field defined.

Reply:
Fixed.